package.json

```diff
@@ -10,6 +10,7 @@
     "dev-client": "yarn clean && yarn prep && webpack --watch --colors --progress",
     "dev-server": "nodemon --watch src/server --watch node_modules src/server/index.js localhost:8080",
     "dev": "concurrently \"yarn dev-client\" \"yarn dev-server\"",
+    "dev-parity": "concurrently \"yarn parity\" \"yarn dev-client\" \"yarn dev-server\"",
```

Comment:
I would run `parity` concurrently in the `"dev"` script. We almost always will want to run parity when developing, yes?

Reply:
we had a configuration like this for testrpc on meteor, sometimes it's better run it separately for better debug. Change as you prefer man! =)

Reply:
Nah that makes sense, 👍 !